docs/contributing/code-style/rust.md

```diff
@@ -0,0 +1,141 @@
+---
+toc_min_heading_level: 2
+toc_max_heading_level: 5
+---
+
+# Rust
+
+We use `rustfmt` to format our code. This tool is part of the Rust toolchain and will format your
+code according to the Rust style guide. We also use `clippy` to catch common mistakes and improve
+the quality of our code.
+
+This document covers some additional guidelines that are not enforced by `rustfmt` or `clippy`. And
+will generally allow you to write more idiomatic Rust code.
+
+## Avoid panics
+
+Panics are forbidden in the Bitwarden codebase outside of tests. Errors should be handled gracefully
```

Comment:
Technically `expect` results in a panic, therefore panics are not forbidden, only `unwrap` is forbidden.

suggestion: 
```suggestion
Panics are highly discouraged in the Bitwarden codebase outside of tests. Errors should be handled gracefully
```